platforms/substrate/configuration/samples/network-example.yaml

```diff
@@ -0,0 +1,277 @@
+##############################################################################################
```

Comment:
@kaverikhaneja , @TonyRowntree , @angelaalagbe : This is a sample configuration file for using with Substrate default. Update all the `changeme` and `CHANGE_ME` and keep in a separate location or build folder.